index.js

```diff
@@ -68,6 +68,9 @@ const optionDefinitions = [
   {
     name: 'repository', alias: 'r', typeLabel: '<username/repository or URL>', description: 'The repository to spellcheck.',
   },
+  {
+    name: 'no-pr', type: Boolean, description: 'Do not create the pull request.',
```

Comment:
How about "Do not automatically create a pull request on the target repository."?